report/report.go

```diff
@@ -151,6 +160,18 @@ type Report struct {
 	// their status endpoints. Edges are present.
 	Overlay Topology
 
+	// Persistent Volume nodes represent all Kubernetes Persistent Volumes running on hosts running probes.
+	// Metadata is limited for now, more to come later.
+	PersistentVolume Topology
+
+	// Persistent Volume Claim nodes represent all Kubernetes Persistent Volume Claims running on hosts running probes.
+	// Metadata is limited for now, more to come later.
+	PersistentVolumeClaim Topology
+
+	// Storage Class represent all kubernetes Storage Classes on hosts running probes.
+	// Metadata is limited for now, more to come later.
+	StorageClass Topology
```

Comment:
Do we really need three topologies to achieve the aim?  Could we just have one "storage" topology that included three kinds of nodes?

Reply:
I am trying to bring everything related to the storage under `storage` topology.  I am giving it a shot.

Reply:
I think there'll be a problem with `shapes` of these components if we put them in the same topology.

Reply:
I haven't looked at that code, but I see other views use multiple shapes in the same view.  What actually goes wrong?